src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      <p className="Person__partner">
+        {isMarried
+          ? (
+            `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
+          ) : (
+            'I am not married'
+          )
+        }
```

Comment:
As for me the ternary inside another ternary looks a little weird. Maybe it would be better to move sex ternary to constant and then use the result of it inside our isMarried ternary?